libcxx/cmake/caches/Generic-fast-hardening-mode.cmake

```diff
@@ -0,0 +1 @@
+set(LIBCXX_HARDENING_MODE "fast" CACHE STRING "")
```

Comment:
You could consider naming these `Generic-hardening-mode-debug.cmake` (or just `Generic-hardening-debug.cmake`) & al so that they show up side by side when browsing files. Not necessary, but might be nice.